robot_smach_states/examples/manipulation/grasp_benchmark/grasping_height_test.py

```diff
@@ -0,0 +1,77 @@
+#! /usr/bin/env python
+
+# System
+from collections import namedtuple
+
+# ROS
+import PyKDL as kdl
+import rospy
+
+# TU/e Robotics
+from robot_smach_states.manipulation import Grab
+from robot_smach_states.navigation import NavigateToSymbolic
+from robot_smach_states.util.designators import EntityByIdDesignator, UnoccupiedArmDesignator
+
+# Robot Skills
+from robot_skills import arms
+from robot_skills.hero_parts.hero_arm import HeroArm
+from robot_skills.get_robot import get_robot_from_argv
+from robot_skills.util.kdl_conversions import FrameStamped
+
+DZ = 0.1
+Z_MAX = 1.2
+
+X_ENTITY = 3.2
+Y_ENTITY = 1.55
+
+Result = namedtuple("Result", ["height", "result", "duration"])
+
+if __name__ == "__main__":
+
+    rospy.init_node("test_grasping")
+
+    robot = get_robot_from_argv(1)
+
+    # Navigate to the couch table
+    nav_designator = EntityByIdDesignator(robot=robot, id="couch_table")
+    nav_state = NavigateToSymbolic(robot, {nav_designator: "in_front_of"}, nav_designator)
+    nav_state.execute()
+
+    # Setup the test
+    z_values = [DZ]
+    while z_values[-1] < Z_MAX:
+        z_values.append(z_values[-1] + DZ)
+
+    results = []
+    for z in z_values:
```

Comment:
Add 'rospy.is_shutdown()` check so that intermediate results will be printed as well

Reply:
Fixed in 3b6ee2de8aaa4948a84d1db49c308fa6b672bb0e